### ISSUE TITLE
[incompability] Supplementaries

### ISSUE THREAD

**Issue by yuma-dev** [MehVahdJukaar/FarmersDelightRefabricated#118]
### Minecraft version

1.21

### Farmer's Delight version

2.1.10

### Fabric Loader version

0.16.9

### Fabric API version

0.102.0

### Description

Caused by: java.lang.NoSuchFieldError: Class vectorwing.farmersdelight.common.Configuration does not have member field 'java.util.function.Supplier ENABLE_TOMATO_VINE_CLIMBING_TAGGED_ROPES'
Something about the tomato vines does not work with supplementaries and makes the server crash at 18:28:06

### Steps to reproduce

1. install supplementaries with delight refabricated
2. plant tomato vines and let them grow i think

### Mod list

[15:52:15] [main/INFO]: Loading 287 mods:
	- accurateblockplacement 1.2.5
	- almanac 1.0.2
	- ambientsounds 6.0.5
	- amendments 1.21-1.2.19
	- animatica 0.6.1+1.21
	- appleskin 3.0.6+mc1.21
	- architectury 13.0.8
	- armorstatues 21.0.1
	   \-- statuemenus 21.0.2
	- athena 4.0.1
	- balm-fabric 21.0.20
	   \-- kuma_api 21.0.5-SNAPSHOT
	- beautify 1.2.1+1.21
	- betterf3 11.0.1
	- bettermounthud 1.2.4
	- betterpingdisplay 1.1.1
	- betterthanmending 2.1.0
	- betterthirdperson 1.9.0
	- capes 1.5.4+1.21
	- chipped 4.0.0
	- cicada 0.9.1+1.21-1.21.1
	- citresewn 1.2.2+1.21
	   \-- citresewn-defaults 1.2.2+1.21
	- clientsort 1.1.1+1.21
	- cloth-config 15.0.140
	   \-- cloth-basic-math 0.6.1
	- collective 7.87
	- comforts 9.0.3+1.21.1
	   |-- cardinal-components-base 6.1.0
	   |-- cardinal-components-entity 6.1.0
	   \-- spectrelib 0.17.2+1.21
	        |-- com_electronwill_night-config_core 3.8.0
	        \-- com_electronwill_night-config_toml 3.8.0
	- continuity 3.0.0-beta.5+1.21
	- controlling 18.0.4
	- creativecore 2.12.9
	   \-- net_neoforged_bus 7.2.0
	- cropsloverain 2.1.1
	- cubes-without-borders 3.0.0+mc1.21
	- debugify 1.21+1.0
	- detailab 2.6.3+1.21-fabric
	- distanthorizons 2.2.1-a
	   |-- fabric-api-base 0.4.42+6573ed8c90
	   |-- fabric-key-binding-api-v1 1.0.47+0af3f5a790
	   \-- fabric-screen-api-v1 2.0.24+b559734490
	- do_a_barrel_roll 3.7.2
	- e4mc_minecraft 5.2.1
	   \-- folk_sisby_kaleido-config 0.3.1+1.3.1
	- easyanvils 21.0.5
	- easymagic 21.0.4
	- easyshulkerboxes 21.0.3
	   \-- iteminteractions 21.0.5
	- elytraslot 9.0.1+1.21.1
	- elytratrims 3.5.4
	- emi 1.1.12+1.21+fabric
	- emi_enchanting 0.1.2+1.21
	- emi_loot 0.7.4+1.21+fabric
	- enhancedblockentities 0.10.2+1.21
	- entity_model_features 2.2.6
	- entity_texture_features 6.2.8
	   \-- org_apache_httpcomponents_httpmime 4.5.10
	- entityculling 1.7.1
	- fabric-api 0.102.0+1.21
	   |-- fabric-api-lookup-api-v1 1.6.68+b5597344d1
	   |-- fabric-biome-api-v1 13.0.29+5bd9f1bcd1
	   |-- fabric-block-api-v1 1.0.22+0af3f5a7d1
	   |-- fabric-block-view-api-v2 1.0.10+6573ed8cd1
	   |-- fabric-blockrenderlayer-v1 1.1.52+0af3f5a7d1
	   |-- fabric-client-tags-api-v1 1.1.15+6573ed8cd1
	   |-- fabric-command-api-v1 1.2.49+f71b366fd1
	   |-- fabric-command-api-v2 2.2.28+6ced4dd9d1
	   |-- fabric-commands-v0 0.2.66+df3654b3d1
	   |-- fabric-content-registries-v0 8.0.16+b5597344d1
	   |-- fabric-convention-tags-v1 2.0.19+7f945d5bd1
	   |-- fabric-convention-tags-v2 2.5.0+c5e2b5c6d1
	   |-- fabric-crash-report-info-v1 0.2.29+0af3f5a7d1
	   |-- fabric-data-attachment-api-v1 1.1.27+6a6dfa19d1
	   |-- fabric-data-generation-api-v1 20.2.16+16c4ae25d1
	   |-- fabric-dimensions-v1 4.0.0+6fc22b99d1
	   |-- fabric-entity-events-v1 1.6.12+6fc22b99d1
	   |-- fabric-events-interaction-v0 0.7.12+ba9dae06d1
	   |-- fabric-game-rule-api-v1 1.0.53+6ced4dd9d1
	   |-- fabric-item-api-v1 11.0.0+afdfc921d1
	   |-- fabric-item-group-api-v1 4.1.4+78017270d1
	   |-- fabric-keybindings-v0 0.2.45+df3654b3d1
	   |-- fabric-lifecycle-events-v1 2.3.12+6c1df360d1
	   |-- fabric-loot-api-v2 3.0.14+3f89f5a5d1
	   |-- fabric-loot-api-v3 1.0.2+3f89f5a5d1
	   |-- fabric-message-api-v1 6.0.13+6573ed8cd1
	   |-- fabric-model-loading-api-v1 2.0.0+fe474d6bd1
	   |-- fabric-networking-api-v1 4.2.2+60c3209bd1
	   |-- fabric-object-builder-api-v1 15.2.0+a551f7a4d1
	   |-- fabric-particles-v1 4.0.2+6573ed8cd1
	   |-- fabric-recipe-api-v1 5.0.12+65089712d1
	   |-- fabric-registry-sync-v0 5.1.2+60c3209bd1
	   |-- fabric-renderer-api-v1 3.4.0+c705a49cd1
	   |-- fabric-renderer-indigo 1.7.0+c705a49cd1
	   |-- fabric-renderer-registries-v1 3.2.68+df3654b3d1
	   |-- fabric-rendering-data-attachment-v1 0.3.48+73761d2ed1
	   |-- fabric-rendering-fluids-v1 3.1.6+b5597344d1
	   |-- fabric-rendering-v0 1.1.71+df3654b3d1
	   |-- fabric-rendering-v1 5.0.5+df16efd0d1
	   |-- fabric-resource-conditions-api-v1 4.3.0+8dc279b1d1
	   |-- fabric-resource-loader-v0 1.3.0+56599129d1
	   |-- fabric-screen-handler-api-v1 1.3.86+b5597344d1
	   |-- fabric-sound-api-v1 1.0.23+6573ed8cd1
	   |-- fabric-transfer-api-v1 5.1.16+3dccd343d1
	   \-- fabric-transitive-access-wideners-v1 6.1.0+176f9036d1
	- fabric-language-kotlin 1.12.3+kotlin.2.0.21
	   |-- org_jetbrains_kotlin_kotlin-reflect 2.0.21
	   |-- org_jetbrains_kotlin_kotlin-stdlib 2.0.21
	   |-- org_jetbrains_kotlin_kotlin-stdlib-jdk7 2.0.21
	   |-- org_jetbrains_kotlin_kotlin-stdlib-jdk8 2.0.21
	   |-- org_jetbrains_kotlinx_atomicfu-jvm 0.25.0
	   |-- org_jetbrains_kotlinx_kotlinx-coroutines-core-jvm 1.9.0
	   |-- org_jetbrains_kotlinx_kotlinx-coroutines-jdk8 1.9.0
	   |-- org_jetbrains_kotlinx_kotlinx-datetime-jvm 0.6.1
	   |-- org_jetbrains_kotlinx_kotlinx-io-bytestring-jvm 0.5.4
	   |-- org_jetbrains_kotlinx_kotlinx-io-core-jvm 0.5.4
	   |-- org_jetbrains_kotlinx_kotlinx-serialization-cbor-jvm 1.7.3
	   |-- org_jetbrains_kotlinx_kotlinx-serialization-core-jvm 1.7.3
	   \-- org_jetbrains_kotlinx_kotlinx-serialization-json-jvm 1.7.3
	- fabricloader 0.16.9
	- fabricskyboxes 0.7.4+mc1.21
	- fabrishot 1.14.1
	- fallingleaves 1.16.2
	- fallingtree 1.21.0.8
	- farmersdelight 1.21-2.1.10+refabricated
	   |-- mm 2.3
	   |-- porting_lib_base 3.1.0-beta+fdrf.1+1.21
	   |    |-- com_electronwill_night-config_core 3.8.0
	   |    |-- com_electronwill_night-config_toml 3.8.0
	   |    |-- mm 2.3
	   |    |-- porting_lib_accessors 3.1.0-beta+fdrf.1+1.21
	   |    |-- porting_lib_asm 3.1.0-beta+fdrf.1+1.21
	   |    |-- porting_lib_attributes 3.1.0-beta+fdrf.1+1.21
	   |    |-- porting_lib_blocks 3.1.0-beta+fdrf.1+1.21
	   |    |-- porting_lib_common 3.1.0-beta+fdrf.1+1.21
	   |    |-- porting_lib_core 3.1.0-beta+fdrf.1+1.21
	   |    |-- porting_lib_entity 3.1.0-beta+fdrf.1+1.21
	   |    |-- porting_lib_extensions 3.1.0-beta+fdrf.1+1.21
	   |    |-- porting_lib_fluids 3.1.0-beta+fdrf.1+1.21
	   |    |-- porting_lib_gametest 3.1.0-beta+fdrf.1+1.21
	   |    |-- porting_lib_gui_utils 3.1.0-beta+fdrf.1+1.21
	   |    |-- porting_lib_mixin_extensions 3.1.0-beta+fdrf.1+1.21
	   |    |-- porting_lib_tags 3.0
	   |    \-- porting_lib_transfer 3.1.0-beta+fdrf.1+1.21
	   |-- porting_lib_config 3.1.0-beta+fdrf.1+1.21
	   |    |-- com_electronwill_night-config_core 3.8.0
	   |    |-- com_electronwill_night-config_toml 3.8.0
	   |    |-- porting_lib_core 3.1.0-beta+fdrf.1+1.21
	   |    \-- porting_lib_gametest 3.1.0-beta+fdrf.1+1.21
	   |-- porting_lib_item_abilities 3.1.0-beta+fdrf.1+1.21
	   |    |-- porting_lib_blocks 3.1.0-beta+fdrf.1+1.21
	   |    |-- porting_lib_core 3.1.0-beta+fdrf.1+1.21
	   |    |-- porting_lib_gametest 3.1.0-beta+fdrf.1+1.21
	   |    \-- porting_lib_level_events 3.1.0-beta+fdrf.1+1.21
	   |-- porting_lib_lazy_registration 3.1.0-beta+fdrf.1+1.21
	   |    |-- porting_lib_core 3.1.0-beta+fdrf.1+1.21
	   |    \-- porting_lib_gametest 3.1.0-beta+fdrf.1+1.21
	   |-- porting_lib_loot 3.1.0-beta+fdrf.1+1.21
	   |    |-- porting_lib_conditions 3.1.0-beta+fdrf.1+1.21
	   |    |-- porting_lib_core 3.1.0-beta+fdrf.1+1.21
	   |    \-- porting_lib_gametest 3.1.0-beta+fdrf.1+1.21
	   \-- porting_lib_recipe_book_categories 3.1.0-beta+fdrf.1+1.21
	        |-- porting_lib_core 3.1.0-beta+fdrf.1+1.21
	        \-- porting_lib_gametest 3.1.0-beta+fdrf.1+1.21
	- fastquit 3.0.0+1.20.6
	- ferritecore 7.0.0
	- forgeconfigapiport 21.0.8
	   |-- com_electronwill_night-config_core 3.8.0
	   \-- com_electronwill_night-config_toml 3.8.0
	- freecam 1.3.0+mc1.21
	- friendsandfoes 3.0.6
	- fsb-interop 1.4.0+mc1.21-build.54
	- fsit 2.6.2+mc1.21
	- fzzy_config 0.5.8+1.21
	   |-- blue_endless_jankson 1.2.3
	   |-- fabric-permissions-api-v0 0.3.1
	   \-- net_peanuuutz_tomlkt_tomlkt-jvm 0.3.7
	- handcrafted 4.0.1
	- highspeed-rail 0.8.1+1.21
	- horsebuff 2.1.8
	- image2map 0.7.1+1.21
	   |-- map-canvas-api 0.4.1+1.21
	   \-- sgui 1.6.0+1.21
	- immediatelyfast 1.3.2+1.21.1
	   \-- net_lenni0451_reflect 1.3.4
	- immersiveui 0.2.2
	- interactic 0.2.3+1.21
	- iris 1.8.0+mc1.21.1
	   |-- io_github_douira_glsl-transformer 2.0.1
	   |-- org_anarres_jcpp 1.4.14
	   \-- org_antlr_antlr4-runtime 4.13.1
	- isxander-main-menu-credits 1.2.0
	- itemswapper 0.7.3
	- jamlib 1.2.0+1.21
	   \-- blue_endless_jankson 1.2.3
	- java 21
	- kaffees_dual_ride 1.1.0
	- kleeslabs 21.0.2
	- krypton 0.2.8
	   \-- com_velocitypowered_velocity-native 3.3.0-SNAPSHOT
	- languagereload 1.7.1+1.21.1
	- letmedespawn 1.4.4
	- lighty 3.0.0-beta.4
	   |-- fabric-api-base 0.4.42+6573ed8c90
	   \-- fabric-key-binding-api-v1 1.0.47+0af3f5a790
	- lithium 0.13.1
	- loyal_items 1.0.2
	- magnumtorch 21.0.0
	- minecraft 1.21
	- mixintrace 1.1.1+1.17
	- modelfix 1.21-1.6
	- modernfix 5.19.1+mc1.21
	- modmenu 11.0.3
	   |-- fabric-api-base 0.4.42+6573ed8c90
	   |-- fabric-key-binding-api-v1 1.0.47+0af3f5a790
	   \-- fabric-screen-api-v1 2.0.24+b559734490
	- moonlight 1.21-2.17.7
	- morechathistory 1.3.1
	- moreculling 1.0.0
	   |-- conditional-mixin 0.6.3
	   \-- mixinsquared 0.2.0
	- mousetweaks 2.26
	- mr_darcenos_minecarts 1.4
	- mr_enchantments_encore 1.8
	- mr_jakes_buildtools 1.6
	- mr_mastercutter 1.3.1+1.21
	- mr_vanilla_refresh 1.4.26
	- mru 1.0.4+1.21+fabric
	- niftycarts 21.0.1
	- nochatreports 1.21.1-v2.9.1
	- octolib 0.4.2
	   \-- org_yaml_snakeyaml 2.2
	- optigui 2.3.0-beta.6+1.21
	   |-- org_apache_commons_commons-text 1.12.0
	   \-- org_ini4j_ini4j 0.5.4
	- owo 0.12.15+1.21
	   |-- io_wispforest_endec 0.1.5
	   |-- io_wispforest_endec_gson 0.1.3
	   |-- io_wispforest_endec_jankson 0.1.3
	   \-- io_wispforest_endec_netty 0.1.2
	- paginatedadvancements 2.5.1
	- pickupnotifier 21.0.0
	- ping-wheel 1.10.0
	- polymer-bundled 0.9.9+1.21
	   |-- polymer-autohost 0.9.9+1.21
	   \-- polymer-blocks 0.9.9+1.21
	- polymorph 1.0.6+1.21.1
	   |-- cardinal-components-base 6.1.0
	   |-- cardinal-components-block 6.1.0
	   \-- cardinal-components-entity 6.1.0
	- polytone 1.21-2.5.12
	   \-- net_objecthunter_exp4j 0.4.8
	- presencefootsteps 1.10.2+1.21
	   \-- kirin 1.20.0+1.21
	- puzzle 2.0.1
	   \-- midnightlib 1.6.3
	- puzzleslib 21.0.28
	- reeses-sodium-options 1.8.0-beta.4+mc1.21.1
	- resourcefullib 3.0.11
	   |-- com_teamresourceful_bytecodecs 1.1.2
	   \-- com_teamresourceful_yabn 1.0.3
	- rightclickharvest 4.3.0+1.21
	- rrls 5.0.7+mc1.21-fabric
	- screenshot_viewer 1.3.3-fabric-mc1.21
	   \-- catconfig-mc 0.2.1
	        \-- io_github_lgatodu47_catconfig 0.3.0
	- screenshotclipboard 1.0.10
	- searchables 1.0.5
	- shulkerplus 1.2.2
	- sodium 0.6.0+mc1.21.1
	- sodium-extra 0.6.0-beta.3+mc1.21.1
	- sound_physics_remastered 1.21.1-1.4.6
	- sounds 2.2.1+1.21+fabric
	   \-- mixinextras 0.5.0-beta.2
	- stickykeys 2.2.7
	- stoneworks 21.0.0
	- supplementaries 1.21-3.0.25-beta
	- suppsquared 1.21-1.2.3
	- taxfreelevels 1.4.1
	- toggleableitemframes 5.1.0-1.21.1
	- trinkets 3.10.0
	   |-- cardinal-components-base 6.1.0
	   \-- cardinal-components-entity 6.1.0
	- universal-graves 3.4.4+1.21
	   |-- common-protection-api 1.0.0
	   |-- placeholder-api 2.4.1+1.21
	   |-- polymer-core 0.9.14+1.21.1
	   |    |-- polymer-networking 0.9.14+1.21.1
	   |    |    \-- polymer-common 0.9.14+1.21.1
	   |    |         \-- packet_tweaker 0.5.5+1.21
	   |    \-- polymer-registry-sync-manipulator 0.9.14+1.21.1
	   |-- polymer-resource-pack 0.9.14+1.21.1
	   |    \-- polymer-common 0.9.14+1.21.1
	   |         \-- packet_tweaker 0.5.5+1.21
	   |-- polymer-virtual-entity 0.9.14+1.21.1
	   |    \-- polymer-common 0.9.14+1.21.1
	   |         \-- packet_tweaker 0.5.5+1.21
	   |-- predicate-api 0.5.1+1.21
	   |-- server_translations_api 2.3.1+1.21-pre2
	   \-- sgui 1.6.0+1.21
	- veinmining 5.0.0-beta+1.21
	- visualworkbench 21.0.5
	- voicechat 1.21.1-2.5.26
	- waveycapes 1.4.12
	- xaerominimap 24.6.1
	- xaeroworldmap 1.39.0
	- yet_another_config_lib_v3 3.6.1+1.21-fabric
	   |-- com_twelvemonkeys_common_common-image 3.10.0
	   |-- com_twelvemonkeys_common_common-io 3.10.0
	   |-- com_twelvemonkeys_common_common-lang 3.10.0
	   |-- com_twelvemonkeys_imageio_imageio-core 3.10.0
	   |-- com_twelvemonkeys_imageio_imageio-metadata 3.10.0
	   |-- com_twelvemonkeys_imageio_imageio-webp 3.10.0
	   |-- org_quiltmc_parsers_gson 0.2.1
	   \-- org_quiltmc_parsers_json 0.2.1
	- yosbr 0.1.2
	- zoomify 2.14.2+1.21.1
	   \-- com_akuleshov7_ktoml-core-jvm 0.5.2

### Logs

net.minecraft.class_148: Exception ticking world
	at knot/net.minecraft.server.MinecraftServer.method_3813(MinecraftServer.java:1025) ~[server-intermediary.jar:?]
	at knot/net.minecraft.class_3176.method_3813(class_3176.java:299) ~[server-intermediary.jar:?]
	at knot/net.minecraft.server.MinecraftServer.method_3748(MinecraftServer.java:912) ~[server-intermediary.jar:?]
	at knot/net.minecraft.server.MinecraftServer.method_29741(MinecraftServer.java:697) ~[server-intermediary.jar:?]
	at knot/net.minecraft.server.MinecraftServer.method_29739(MinecraftServer.java:281) ~[server-intermediary.jar:?]
	at java.base/java.lang.Thread.run(Thread.java:1583) [?:?]
Caused by: java.lang.NoSuchFieldError: Class vectorwing.farmersdelight.common.Configuration does not have member field 'java.util.function.Supplier ENABLE_TOMATO_VINE_CLIMBING_TAGGED_ROPES'
	at knot/net.mehvahdjukaar.supplementaries.integration.fabric.FarmersDelightCompatImpl.isTomatoVineClimbingConfigOn(FarmersDelightCompatImpl.java:7) ~[supplementaries-1.21-3.0.25-beta-fabric.jar:?]
	at knot/net.mehvahdjukaar.supplementaries.integration.FarmersDelightCompat.isTomatoVineClimbingConfigOn(FarmersDelightCompat.java) ~[supplementaries-1.21-3.0.25-beta-fabric.jar:?]
	at knot/net.mehvahdjukaar.supplementaries.integration.FarmersDelightCompat.getTomatoLoggedReplacement(FarmersDelightCompat.java:76) ~[supplementaries-1.21-3.0.25-beta-fabric.jar:?]
	at knot/vectorwing.farmersdelight.common.block.TomatoVineBlock.handler$dih000$supplementaries$supp$tomatoLoggingCompat(TomatoVineBlock.java:534) ~[FarmersDelight-1.21-2.1.10+refabricated.jar:?]
	at knot/vectorwing.farmersdelight.common.block.TomatoVineBlock.attemptRopeClimb(TomatoVineBlock.java:97) ~[FarmersDelight-1.21-2.1.10+refabricated.jar:?]
	at knot/vectorwing.farmersdelight.common.block.TomatoVineBlock.method_9514(TomatoVineBlock.java:91) ~[FarmersDelight-1.21-2.1.10+refabricated.jar:?]
	at knot/net.minecraft.class_4970$class_4971.method_26199(class_4970.java:1153) ~[server-intermediary.jar:?]
	at knot/net.minecraft.class_3218.redirect$bob000$lithium$redirectBlockStateTick(class_3218.java:6289) ~[server-intermediary.jar:?]
	at knot/net.minecraft.class_3218.method_18203(class_3218.java:519) ~[server-intermediary.jar:?]
	at knot/net.minecraft.class_3215.method_14161(class_3215.java:365) ~[server-intermediary.jar:?]
	at knot/net.minecraft.class_3215.method_12127(class_3215.java:310) ~[server-intermediary.jar:?]
	at knot/net.minecraft.class_3218.method_18765(class_3218.java:350) ~[server-intermediary.jar:?]
	at knot/net.minecraft.server.MinecraftServer.method_3813(MinecraftServer.java:1021) ~[server-intermediary.jar:?]
	... 5 more

Server thread/ERROR
18:28:06

### Minimal instance

- [ ] I have tested this on a minimal instance

### Performance and shader mods

- [X] I am using performance or shader mods

**Comment by Poopooracoocoo** [comment]
Can you reproduce with the latest version of FDR? https://modrinth.com/mod/farmers-delight-refabricated/versions?g=1.21.1&g=1.21&l=fabric

You can either update to 1.21.1 (fixed only two bugs) or create a [dependency override](https://wiki.fabricmc.net/tutorial:dependency_overrides)

**Comment by yuma-dev** [comment]
I can't reproduce the error on 1.21.1, also cannot update to 1.21.1 as some of the mods in the modpack don't exist in 1.21.1
As for the dependency override, I am not savvy enough with fabric to understand what I would need to change or how to change it
I can reproduce the error on 1.21 with the latest versions on a minimal instance tho

**Comment by ArmoredFuzzball** [comment]
I can confirm after updating to the latest (as of 14 hours ago, 2.2.3 (you didn't bump the version btw)) on 1.21.1, the same issue persists with a basically identical stack trace.

**Comment by MerchantPug** [comment]
Supplementaries issue. Seems like a config option had its name modified as of 1.2.6.

ENABLE_TOMATO_VINE_CLIMBING_TAGGED_ROPES -> ENABLE_TOMATO_VINE_CLIMBING_TAGGED_ROPES

@MehVahdJukaar

**Comment by ChrysanthCow** [comment]
https://github.com/MehVahdJukaar/FarmersDelightRefabricated/issues/106

Tl;dr, you have an addon loading an old version of Porting Lib.

**Comment by yuma-dev** [comment]
linked issue is telling me to search for "3.1.0-beta+fdrf.1+1.21" which was used before updating to 1.21.1 and now after updating its 3.1.0-fdrf.4+1.21.1
still, the same issue persists with tomatoes crashing the server/client

**Comment by ChrysanthCow** [comment]
Not a Supplementaries issue. This was caused by a faulty merge commit.

**Comment by yuma-dev** [comment]
I see. Does that mean it's not an Issue on my end and will be fixed in an Update?

**Comment by ChrysanthCow** [comment]
Yes, it'll be fixed in the next update.

**Comment by ChrysanthCow** [comment]
Fixed in 2.2.5!
